app.json

```diff
@@ -19,7 +19,7 @@
     "DYNAMODB_REGION": "eu-west-3",
     "DYNAMODB_TABLE_NAME": "fewlines-connect-account-review",
     "IS_JWE_SIGNED": "true",
-    "NEXT_PUBLIC_SENTRY_DSN": "https://6baf0213093a428d9e9ec7ae0979868c@o287463.ingest.sentry.io/5387583",
+    "SENTRY_AUTH_TOKEN": "e010c0ab20964317bed616fed74e2a5e92157c919399452ea844d11989b143ab",
```

Comment:
so what is it now? A public token, a PAT? Where are we ? 😁

Reply:
Looks like I did not push my last changes before leaving.
This will be a PAT, and I removed this line as it will be injected thoughts Heroku secrets.